packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -415,22 +380,24 @@ function WorklistDataGrid(props) {
    */
   const moveFocusFromGrid = (moveForward) => {
     // add all elements we want to include in our selection
-    const focusableElements = [...document.body.querySelectorAll(
-      'a[href], button, input, textarea, select, details,[tabindex]:not([tabindex="-1"]',
-    )].filter(
+    const focusableElementSelector = 'a[href]:not([tabindex=\'-1\']), area[href]:not([tabindex=\'-1\']), input:not([disabled]):not([tabindex=\'-1\']), '
```

Comment:
Wonder if there's a simpler way to specify this selector, or not have to rely on using the query selector for this?

Reply:
I have to find the focusable elements in the DOM.  Unfortunately, there is no built-in way to get a list of focusable elements.